src/svgutils/common.py

```diff
@@ -0,0 +1,59 @@
+import re
```

Comment:
I manually verified locally that this new file contains an unmodified copy of class `Unit` because withouzt that it was not at all clear. That's something that ideally a dedicated commit a la "Move class Unit to a dedicated module" would clearly communicate in Git history. (I'd personally do that on a new temp branch and then rebase this PR on top of it, but there are other ways.) Just an idea.

Reply:
yes, the class is exactly the same. I needed to move it to separate module to avoid circular imports.

Reply:
done